backend/app/services/flow_scheduler.py

```
@@ -0,0 +1,389 @@
+def _trigger_task_execution(db, flow: FlowResource, execution) -> None:
+    """
+    Trigger the actual task execution for a flow.
+
+    This creates a Task resource and links it to the flow execution.
+    The task will be executed by the existing task execution system.
+    """
+    try:
+        from app.models.task import TaskResource
+        from app.models.kind import Kind
+
+        flow_crd = Flow.model_validate(flow.json)
```

Ruff (0.14.10)

Local variable `flow_crd` is assigned to but never used

Remove assignment to unused variable `flow_crd`

(F841)